authentification/urls.py

```diff
@@ -3,7 +3,7 @@
 from . import views
```

Comment:
It is better to import individual views rather than importing complete view file.